### ISSUE TITLE
Build error on Ubuntu 15.05

### ISSUE THREAD

**Issue by Danappelxx** [takebayashi/http4swift#1]
I ran `swift build` after cloning swiftra-example and got the following error:

```
$ swift build
Cloning Packages/swiftra
Cloning Packages/http4swift
Compiling Swift Module 'http4swift' (6 sources)
/home/dan/Downloads/swiftra-example/Packages/http4swift-0.0.2/Sources/HTTPRequest.swift:87:40: error: value of type 'String' has no member 'hasPrefix'
                    if line.isEmpty || line.hasPrefix(String(HTTPRequestParser.CR)) {
                                       ^~~~ ~~~~~~~~~
<unknown>:0: error: build had 1 command failures
swift-build: exit(1): ["/usr/bin/swift-build-tool", "-f", "/home/dan/Downloads/swiftra-example/.build/debug/http4swift.o/llbuild.yaml"]
```

**Comment by takebayashi** [comment]
Fixed in e4fb9cc.

**Comment by Danappelxx** [comment]
:)
